samples/bluetooth/mesh_provisioner/boards/nrf5340dk_nrf5340_cpuapp_ns.conf

```diff
@@ -1,3 +1,8 @@
+# The option adds TinyCrypt based bt_rand.
+CONFIG_BT_HOST_CRYPTO=n
+# The option adds GATT caching feature that is based on TinyCrypt.
+CONFIG_BT_GATT_CACHING=n
```

Comment:
Is `CONFIG_BT_GATT_CACHING=n` needed here? mesh_provisioner doesn't enable neither peripheral nor central role support.